.gitignore

```diff
@@ -28,3 +28,6 @@ build/
 
 ### VS Code ###
 .vscode/
+
+### SSL Key ###
+/src/main/resources/fixiekey.p12
```

Comment:
end every file with a new line, i pretty sure that IntelliJ warns about that.

Reply:
0 warnings.